jolie/src/main/java/jolie/CommandLineParser.java

```diff
@@ -94,7 +94,7 @@
 	 * 
 	 * @return the arguments passed to the JOLIE program.
 	 */
-	public final String[] arguments() {
+	private final String[] arguments() {
```

Comment:
Do you still need all these private accessor methods? You could just access the fields directly when you create the InterpreterParameters object, since they're private now.